Source/UI/Theme/Generic/Theme+Button.swift

```diff
@@ -53,12 +53,12 @@ public extension Theme {
     func buildNavigationButton(text: String) -> NavigationButtonStyle {
         NavigationButtonStyle(
             borderStyle: ThemeBorderStyle(cornerRadius: self.textInputBorderRadius,
-                                         borderWidth: self.textInputBorderRadius,
-                                         normalColor: self.textInputBorderColor,
-                                         focusColor: self.focussedTextInputBorderColor,
-                                         errorColor: self.errorBorderColor,
-                                         edges: .all,
-                                         corners: nil),
+                                          borderWidth: self.textInputBorderWidth,
```

Comment:
Whats the reason to replace `textInputBorderRadius` with `textInputBorderWidth` ?

Reply:
Previously we were wrongly using `textInputBorderRadius` for both `borderWidth` and `cornerRadius`.

Just linking correctly the source for the border width (which should be `textInputBorderWidth`, not `textInputBorderRadius`)